golang/internal/regression.go

```diff
@@ -0,0 +1,16 @@
+package internal
```

Comment:
Where is this used?

Reply:
regression.go is used for the prediction inside GetAccumulatedProducts(...)